examples/.cargo/riscv32im-ceno-zkvm-elf.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
in future, when users run `ceno-cli init-project ...`,   will this `riscv32im-ceno-zkvm-elf.json` be put into their workspace, (allow editting), or actually this file is not something they can change, bounded to "ceno plaform"?

for the latter case, this file should be in `rt` folder instead of `example` folder?

Reply:
We do not expect users to edit this file.  (And I need to add more documentation for how I made this file, too.  Because the format can change slightly over time as the Rust compiler and llvm evolve.  In fact, it changed already over the last year.)

Yes, something like the `ceno_rt` folder makes conceptual sense.  The main thing is that `examples/.cargo/config.toml` needs to refer to this file by location.

I moved it to where you suggested.

Reply:
The problem that I see is that I don't know how a standalone guest program would refer to `riscv32im-ceno-zkvm-elf.json` that lives in `ceno_rt`, instead of in its the guest's own repository.